test_scripts/PROJECTION/000_Start_secured_audio_service.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@GetmanetsIrina General comment: In this review there is only one comment per issue. Please make sure the same ones are corrected in other places/scripts.